addons/core/functions/fnc_getLrRadioCode.sqf

```diff
@@ -22,4 +22,6 @@
     (call TFAR_fnc_activeLrRadio) call TFAR_fnc_getLrRadioCode;
 */
 
+If (GVARMAIN(radioCodesDisabled)) exitWith {""};
```

Comment:
I think this might be called before CBA settings are initialized.
getLrSettings should not initialize till CBA init is done. Don't know what it should do in that case though.. Maybe return empty array...

Reply:
I don't think this has a bigger impact then you could be able to hear something while you are joining.
If I see this right, the result of this function is nowhere stored for a long time.

Reply:
It is stored in the radio settings array.